### PR TITLE
Update translation.rst

### PR DESCRIPTION
**Pull request by yceruto**
Promoting the new convention.

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
I didn't know this! It's very nice. Thanks.

File: translation.rst [hunk 1]
```diff
@@ -351,9 +351,7 @@ Translation Resource/File Names and Locations
 Symfony looks for message files (i.e. translations) in the following default locations:
 
 * the ``translations/`` directory;
-
-* the ``src/Resources/<bundle name>/translations/`` directory;
-
+* the ``translations/bundles/<bundle name>/`` directory;
```
Comment (xabbuh) [inline translation.rst]: Are you sure about this?

Comment (yceruto) [reply translation.rst]: https://github.com/symfony/symfony/blob/4f9dc9d44988448441da48269793e7cd3fa4b949/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L898-L900

However, the path was wrong (without `bundles/`), thanks! Fixed!

**Review by xabbuh — APPROVED** [review]

File: translation.rst [hunk 2]
```diff
@@ -351,9 +351,7 @@ Translation Resource/File Names and Locations
 Symfony looks for message files (i.e. translations) in the following default locations:
 
 * the ``translations/`` directory;
-
-* the ``src/Resources/<bundle name>/translations/`` directory;
-
+* the ``translations/<bundle name>/`` directory;
```
Comment (weaverryan) [inline translation.rst]: Is there any significance to this? What I mean is: of course `translations/<bundle name>/` is a valid path, as (I assume) that all files are loaded from the `translations/` directory recursively. So, is there anything special about this directory? Translations in this directory already override the ones from the bundle, so I wasn't sure.

Thanks!

Comment (yceruto) [reply translation.rst]: Yep, you're right, translation resources are overridden by domain, so I guess this line is useless, as well as [its definition](https://github.com/symfony/symfony/blob/020664e1efbf8a81dbcc0f7b9dcbfa9ac58c0ed7/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L1157-L1159).

**Comment by yceruto** [comment]
Closing here for the reason explained by @weaverryan https://github.com/symfony/symfony-docs/pull/8857#discussion_r156552014

File: translation.rst (hunk 2 above)
Comment (yceruto) [reply translation.rst]: Removing useless definition https://github.com/symfony/symfony/pull/25476 in core.

**Pull request closed without merging**